packages/veritone-widgets/src/shared/BaseStory.js

```diff
@@ -18,7 +18,8 @@ import {
 } from '../widgets/Notifications';
 
 const app = VeritoneApp({
-  applicationId: '7ef232e8-7c4e-46e0-b9c3-ba53ed9851c1'
+  applicationId: '7ef232e8-7c4e-46e0-b9c3-ba53ed9851c1',
+  isDiscovery: true
```

Comment:
Move this flag to the specific story that renders settings button, hide it by default as none of the other apps will be using it.